src/units/units_conversion.c

```diff
@@ -0,0 +1,147 @@
+/*
+ * Race Capture Firmware
+ *
+ * Copyright (C) 2017 Autosport Labs
+ *
+ * This file is part of the Race Capture firmware suite
+ *
+ * This is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details. You should
+ * have received a copy of the GNU General Public License along with
+ * this code. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "units_conversion.h"
+
+static float no_conversion(float value)
+{
+		return value;
+}
+static float c_to_f(float value)
+{
+		return  value * 1.8 + 32;
+}
+
+static float f_to_c(float value)
+{
+		return (value - 32) / 1.8;
```

Comment:
A thought here: since div is much slower than mult consider making this `(value - 32) * 0.555555556`